afs-storage-api/src/main/java/com/powsybl/afs/storage/Utils.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.afs.storage;
+
+import com.google.common.io.ByteStreams;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.stream.Stream;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipInputStream;
+import java.util.zip.ZipOutputStream;
+
+/**
+ * Utility class.
+ *
+ * @author Valentin Berthault <valentin.berthault at rte-france.com>
+ */
+public final class Utils {
```

Comment:
What not using com.powsybl.commons.compress.ZipPackager ? Is seems to be similar?

Reply:
It does not seem to handle directory compression, does it ?